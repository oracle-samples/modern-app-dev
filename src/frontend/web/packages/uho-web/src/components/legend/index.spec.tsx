/*
 * Copyright (c) 2023 Oracle and/or its affiliates.
 * Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracl.com/licenses/upl/
 */
import { Legend } from '.';
import { h } from 'preact';
import { snapshot } from '../../../test/jetHelper';

snapshot(
  'Legend',
  <Legend
    elements={[
      { color: 'red', text: 'Foo' },
      { color: 'green', text: 'Bar' }
    ]}
    maxHeight={120}
    maxWidth={100}
  />
);
