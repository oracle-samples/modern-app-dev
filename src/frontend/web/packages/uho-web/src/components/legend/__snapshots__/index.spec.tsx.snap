// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot tests for Legend Snapshot: Component - Legend 1`] = `
<DocumentFragment>
  <div
    data-oj-binding-provider="preact"
  >
    <oj-legend
      class="oj-dvtbase oj-legend oj-component-initnode oj-complete"
    >
      <template
        slot="itemTemplate"
      />
      <div
        style="visibility: hidden;"
      />
      <svg
        height="100%"
        width="100%"
      >
        <defs />
        <g
          cursor="default"
          font-family="-apple-system,BlinkMacSystemFont,'Segoe UI','Helvetica Neue',Arial,sans-serif"
          font-size="12px"
          font-weight="400"
        >
          <g>
            <g>
              <rect
                fill="rgba(0,0,0,0)"
                height="120"
                width="100"
              />
              <g>
                <g />
              </g>
            </g>
          </g>
        </g>
      </svg>
    </oj-legend>
  </div>
  <div
    id="__oj_layerhost_container"
  />
</DocumentFragment>
`;
